doc/classes/FileAccess.xml

```diff
@@ -324,6 +324,13 @@
 				Returns [code]null[/code] if opening the file failed. You can use [method get_open_error] to check the error that occurred.
 			</description>
 		</method>
+		<method name="resize">
+			<return type="int" enum="Error" />
+			<param index="0" name="length" type="int" />
+			<description>
+				Resizes the file to a specified length. The file must be open in a mode that permits writing. If the file is extended, NUL characters are appended.
```

Comment:
The Windows docs add the following remark:
> If the file is truncated, all data from the end of the shortened file to the original length of the file is lost.

A similar "this is a destructive operation" remark may be warranted here IMO. This is obvious to developers with experience, but may not be immediately obvious to everyone.